### ISSUE TITLE
Maintenance dornes can unbuckle people from objects (at least the pimpin ride)

### ISSUE THREAD

**Issue by Landerlow** [ParadiseSS13/Paradise#17677]
**Issue Description**:

Maintenance dornes can unbuckle people from objects (at least the pimpin ride)

**What did you expect to happen**: 

They're drones, they shouldn't be able to do that

**What happened instead**:

See above

**Why is this bad/What are the consequences**:

See above. It's not a priority.

**Steps to reproduce the problem**:

Be drone. Unbuckle. 
**When did the problem start happening**:

Noticed it just now
**Extra information**:

N/A

**Comment by GDNgit** [comment]
> > They're drones, they shouldn't be able to do that
> 
> Can you elaborate on that? Why exactly not? Lack of hands (but regular borgs can unbuckle despite that)? Laws violation? (but what about emagged drones?) Small size? (but i'm not sure how would that prevent unbuckling IC'ly.. maybe because i'm not super sure what buckling _means_ ICly..) Just plain being a drone?
> 
> Serious question.

Quick rundown on drones and their functions:
A drone's primary purpose is to perform minor station repairs without interfering with any crew(Not having a noticeable change to the round balance). For example, a drone may choose to repair an airlock in a remote area, let's say teleporter, if someone said "drone stop" the drone would stop and continue on with other tasks at hand. They also will not fight other antagonists the crew may fight (Ex: Blob). The issue with allowing drones to unbuckle people is that a drone could unbuckle a cuffed person which would allow them to escape more quickly, or allow for someone to escape at all for that matter. 

Buckling refers to any act in game that would add/remove the buckle icon in the top right corner of your UI (Ex: Buckling someone to a chair). Buckled people also cannot be pulled.

TLDR: Allows for interference with crew

**Comment by SteelSlayer** [comment]
Yeah this is an issue. It appears that literally any `mob/living` can buckle people and there is currently no way to forbid them from doing so. This would need a refactor for such a system.
